.github/workflows/ci.yml

```
@@ -5,26 +5,57 @@ on:
-  deploy:
+  release-and-deploy:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
       - name: Configure Git Credentials
         run: |
           git config user.name github-actions[bot]
           git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)